common/app/views/fragments/commercial/badge.scala.html

```diff
@@ -1,8 +1,24 @@
 @(content: model.ContentType)(implicit request: RequestHeader)
 @import common.Edition
-@import conf.switches.Switches.SponsoredSwitch
+@import conf.switches.Switches.{SponsoredSwitch, staticBadgesSwitch}
 @import model.{Article, Audio, Gallery, Interactive, Video}
 
+@if(staticBadgesSwitch.isSwitchedOn) {
+
+    @for(branding <- content.branding(Edition(request))) {
+        <div class="ad-slot--paid-for-badge ad-slot--paid-for-badge--article ad-slot--force-display">
```

Comment:
Funnily enough I just pushed change which is renaming this thing to `brandbadge` component. However it is mostly just renaming for now. We will push this out next week and I will do deeper refactoring of the client side of the badge when we will have more badges served from server because as with most of our components its tentacles are spread across the multiple files across the whole codebase.